fetchai/ledger/api/bootstrap.py

```diff
@@ -0,0 +1,50 @@
+import requests
+
+class NetworKUnavailableError(Exception)
+
+def server_from_name(network):
+    # Get list of active servers
+    servers_response = requests.get('https://bootstrap.fetch.ai/networks/')
+    if servers_response.status_code != 200:
+        raise requests.ConnectionError('Failed to get network status from bootstrap')
+
+    # Check requested server is on list
+    available_servers = [s['name'] for s in servers_response.json()]
+    if network not in available_servers:
+        raise NetworKUnavailableError('Requested network not present on network: {}'.format(network))
+
+    # Check network version
+    server_version = next(s for s in servers_response.json() if s['name'] == network)['versions']
+    # TODO: Not sure how to compare this string to our requirements
```

Comment:
I think that you can split the response on `','` and then should be able to use `semver` for this I think

Reply:
(it then can be compared against the `0.8` version)

Reply:
So in this case we're comparing the single __version__ of the API to the version range of the network, whereas on connecting we compare the __compatibility__ range of the API to the single version of the ledger?

What actually is the 'versions' field of the network declaring?